### PR TITLE
feat: added method to validate the ssh connection

### PR DESCRIPTION
**Pull request by zFernand0**
**What It Does**

- Implements an SSH connection validation method
- Remove extra `$ ` from the output of ssh commands at the sdk level

**How to Test**

- Running `zowe uss issue ssh "ls /"`should display all folders/files with proper alignment

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [ ] added/updated automated tests
- [ ] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

Related to:
- https://github.com/zowe/zowe-explorer-vscode/pull/3079

### CODE REVIEW

File: packages/zosuss/src/Shell.ts [hunk 1]
```diff
@@ -164,6 +171,14 @@ export class Shell {
         return this.executeSsh(session, cwdCommand, stdoutHandler);
     }
 
+    public static async isConnectionValid(session: SshSession): Promise<boolean>{
```
Comment (awharn) [inline packages/zosuss/src/Shell.ts:174]: Can we have a system test that tests this please? :)

Comment (zFernand0) [reply packages/zosuss/src/Shell.ts:174]: see e95f9fd41951ca5f1968fca5b6ad167c4816f755

File: packages/zosuss/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zowe z/OS USS SDK package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Removed unnecessary `$ ` characters in front of most output. [zowe-explorer#3079(comment)](https://github.com/zowe/zowe-explorer-vscode/pull/3079#pullrequestreview-2408842655)
+- Enhancement: Added the ability to validate of an SSH profile is able to establish a connection. [zowe-explorer#3079(comment)](https://github.com/zowe/zowe-explorer-vscode/pull/3079#discussion_r1825783867)
```
Comment (gejohnston) [inline packages/zosuss/CHANGELOG.md:8]: I think that Ana will have some feedback on this changelog entry.

**Review by gejohnston — APPROVED** [review]

**Review by traeok — APPROVED** [review]
LGTM, thanks Fernando!

**Comment by zFernand0** [comment]
This should be on public NPM after this workflow is done:
- https://github.com/zowe/zowe-cli-standalone-package/actions/runs/11687950797

**Pull request merged**